deployments/operator/otel-tracing/validation.py

```
@@ -0,0 +1,42 @@
+
+    # Parse resource attributes into a dictionary
+    attrs_dict = {}
+    for attr in resource_attributes.split(","):
+        if "=" in attr:
+            key, value = attr.split("=", 1)
+            key = key.strip()
+            value = value.strip()
+            if not value:
+                raise ValueError(
+                    f"Empty value for attribute '{key}' in AMP_TRACE_ATTRIBUTES"
+                )
```

Ruff (0.14.8)

Avoid specifying long messages outside the exception class

(TRY003)